src/plugins/discover/public/application/helpers/get_sharing_data.test.ts

```diff
@@ -60,7 +58,7 @@ describe('getSharingData', () => {
             },
             "script_fields": Object {},
             "sort": Array [],
-            "stored_fields": Array [],
```

Comment:
This changed to `undefined` since we now exclude `stored_fields` from the request if it's empty.